apps/public-docsite-v9/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -141,6 +141,9 @@ export const FluentDocsPage = () => {
   const dir = primaryStoryContext.parameters?.dir ?? primaryStoryContext.globals?.[DIR_ID] ?? 'ltr';
   const selectedTheme = themes.find(theme => theme.id === primaryStoryContext.globals![THEME_ID]);
 
+  const hideArgsTable = primaryStoryContext.parameters?.docs?.hideArgsTable ?? false;
```

Comment:
- are these official SB apis ? 
- seems this PR does way beyond adding documentation. please update PR and title accordingly

Reply:
parameters is official sb api :) they are normally used to define the behavior of some Storybook features or Storybook addons, but also can be used to pass any arbitrary data